### PR TITLE
Rename Placeholders to Virtual Files in code 

### PR DESCRIPTION
**Pull request by ckamm**
See #6531 

The problem with this is that it needs to be tested from scratch. I tried not to break the nsis installer or the osx double-click work but might have. I didn't see the place where we made the virtual files runnable on linux during my rename spree, so this also needs to be double-checked.

### CODE REVIEW

File: cmake/modules/NSIS.template.in [hunk 1]
```diff
@@ -7,8 +7,8 @@
 !define APPLICATION_CMD_EXECUTABLE "@APPLICATION_EXECUTABLE@cmd.exe"
 !define APPLICATION_DOMAIN "@APPLICATION_DOMAIN@"
 !define APPLICATION_LICENSE "@APPLICATION_LICENSE@"
-!define APPLICATION_PLACEHOLDER_SUFFIX "@APPLICATION_PLACEHOLDER_SUFFIX@"
-!define APPLICATION_PLACEHOLDER_FILECLASS "@APPLICATION_EXECUTABLE@.@APPLICATION_PLACEHOLDER_SUFFIX@"
+!define APPLICATION_VIRTUAL_FILE_SUFFIX "@APPLICATION_VIRTUAL_FILE_SUFFIX@"
```
Comment (ogoffart) [inline cmake/modules/NSIS.template.in]: sometimes there is an underscore, and sometimes not

Comment (ckamm) [reply cmake/modules/NSIS.template.in]: fixed!

**Comment by ckamm** [comment]
However, when I compiled from scratch just now the suffix ended up being `.ownCloud_virtual` and not what was intended.

**Comment by ogoffart** [comment]
@jnweiger  this change again the cmake variable name used in the themes

**Pull request merged**